.travis.yml

```diff
@@ -1,38 +1,43 @@
-language: python
+language: c
 
-python:
-    - 2.7
+os:
+    - osx
+    - linux
 
-env: DISPLAY=:99 MOZ_XVFB=1
+compiler:
+    - gcc
+
+env:
+    - DISPLAY=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo ':99'; fi)
+      MOZ_XVFB=$(if [[ $TRAVIS_OS_NAME = 'linux' ]]; then echo 1; fi)
+      BINARY=$(if [[ $TRAVIS_OS_NAME = 'osx' ]]; then echo 'FirefoxNightly.app/Contents/MacOS/firefox'; else echo 'firefox/firefox'; fi)
```

Comment:
Ok, then lets figure it out later how to better do it. It's certainly not a blocker right now to get rid of this env variable right now.